src/pytest_mock/plugin.py

```diff
@@ -86,9 +86,8 @@ def spy(self, obj: object, name: str) -> unittest.mock.MagicMock:
         Create a spy of method. It will run method normally, but it is now
         possible to use `mock` call features with it, like call count.
 
-        :param object obj: An object.
-        :param unicode name: A method in object.
-        :rtype: unittest.mock.MagicMock
+        :param obj: An object.
```

Comment:
do we need this either?

![image](https://user-images.githubusercontent.com/413772/91635565-65a62980-e9f1-11ea-9460-9609439c1ef2.png)

Reply:
Probably not hehehe